src/de/raw.rs

```diff
@@ -648,13 +648,20 @@ impl<'d, 'de> serde::de::Deserializer<'de> for DocumentKeyDeserializer<'d, 'de>
         }
     }
 
+    fn deserialize_newtype_struct<V>(self, _name: &'static str, visitor: V) -> Result<V::Value>
```

Comment:
This was needed to support deserializing the `Cow` wrappers used in deserialization now. It just forwards deserialization to the wrapped type.